### PR TITLE
Refactor for future maintainability

### PR DESCRIPTION
**Pull request by martialblog**
- Add tests and linting
- Remove `request` requirement by using Python's urllib
- More robust error handling and output generation
- General Refactor for smoother code :trumpet:

### CODE REVIEW

**Review by RincewindsHat — APPROVED** [review]
Looks sane to me on a glance

File: check_brevisone [hunk 1]
```diff
@@ -136,54 +133,88 @@ if __name__ == '__main__':
...
+        base_url = args.protocol + '://' + args.hostname
+        data = get_data(base_url=base_url,
+                        timeout=args.timeout,
+                        insecure=args.insecure)
+    except Exception as data_exc: # pylint: disable=broad-except
+        print('UNKNOWN - Could not connect to SMS Gateway', data_exc)
+        return UNKNOWN
+
+    # Split up lines and clean up items
+    lines = [str(i).strip() for i in data.split("\n") if i]
```
Comment (martialblog) [inline check_brevisone:207]: The printing of all lines should remain. I figured that there might be other that's not perfdata

Comment (martialblog) [reply check_brevisone:207]: Done :heavy_check_mark:

**Comment by martialblog** [comment]
```
python -m coverage report -m --include check_brevisone.py
Name              Stmts   Miss  Cover   Missing
-----------------------------------------------
check_brevisone      79      5    94%   85-86, 199-201
-----------------------------------------------
TOTAL                79      5    94%
```

Good enough

**Pull request merged**